app/assets/stylesheets/mortgage_calculator/components/_mortgagecalc.scss

```diff
@@ -93,6 +93,13 @@
   font-size: px(40);
 }
 
+@media (max-width: 425px){
```

Comment:
* 425px feels a bit magic/overly specific. Can we change the style of the button everywhere so this doesn't happen?
* Style comment: we have spaces before the `{`.
* What's the reason for `white-space: normal;` here?